src/main/java/nextstep/optional/Expression.java

```diff
@@ -1,8 +1,11 @@
 package nextstep.optional;
 
+import java.util.Arrays;
+
 enum Expression {
     PLUS("+"), MINUS("-"), TIMES("*"), DIVIDE("/");
 
+    public static final String EXPRESSION_ERROR_MESSAGE = "%s는 사칙연산에 해당하지 않는 표현식입니다.";
```

Comment:
```suggestion
    private static final String EXPRESSION_ERROR_MESSAGE = "%s는 사칙연산에 해당하지 않는 표현식입니다.";
```

private으로 해도 괜찮지 않을가요?